test/promql/tests/data_plane/gitops_sync_test.yaml

```
@@ -0,0 +1,84 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.gitops_sync_alerts.yaml
+
+tests:
+  # Alerted cases:
+  - interval: 1m
+    input_series:
+
+      # GitOps service is experiencing application sync failure rate of less than 95%, so alert
```

Should say less than 95% success. No?
```suggestion
      # GitOps service is experiencing application sync success rate of less than 95%, so alert
```